Schema/FeatureFlag.v2.0.0.schema.json

```diff
@@ -144,7 +144,7 @@
                   "type": "string",
                   "title": "Variant Name",
                   "description": "The name used to refer to a feature variant.",
-                  "pattern": "^(.*)$"
+                  "pattern": "^[^:\n\r%]*$"
```

Comment:
Do we know if there is a problem for the variant name to contain `:` or `%`?

Reply:
If we're moving that restriction, the pattern should just be "^.*$"

Reply:
Interesting. I didn't know that worked. Good call. Thank you.